packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -165,7 +167,8 @@ const metadata: Meta<SelectArgs> = {
         dropDownPosition: 'below',
         appearance: DropdownAppearance.underline,
         optionsType: ExampleOptionsType.simpleOptions,
-        placeholder: false
+        placeholder: false,
+        clearable: false
```

Comment:
Can you add a description for the new attribute (and existing ones that don't have one yet if you're feeling ambitious)?